docs/visualize_smpl.md

```diff
@@ -201,7 +202,7 @@
 
 - **body model:**
     There area two ways to pass body model:
-    1). You pass `model_path`, `model_type`(optional) and `gender`(optional).
+    1). You pass a dict `body_model_config` which containing the same configs as build_body_model
     2). You pass `body_model` directly and the above three will be ignored.
```

Comment:
`and the dict above will be ignored.`
In the next line (A & B & C) can be replaced by `body_model_config`.

Reply:
ok